docs/src/operate/c8y/connect.md

```diff
@@ -58,183 +59,236 @@ in this case we are told to run `tedge config set c8y.url <value>`.
 
 ## Making the cloud trust the device
 
-The next step is to have the device certificate trusted by Cumulocity. This is done by uploading the certificate of the signee.
-You can upload the root certificate using the [Cumulocity UI](https://cumulocity.com/docs/device-management-application/managing-device-data/#managing-trusted-certificates)
-or with [`tedge cert upload`](../../references/cli/tedge-cert.md) command as described below.
+%%te%% primarily uses a x509 certificate to connect to Cumulocity, therefore the certificate must be trusted by Cumulocity in order for the connection to be established.
 
-:::note
-The `tedge cert upload` command requires the credentials of a Cumulocity user
-having the permissions to upload trusted certificates on the Cumulocity tenant of the device.
-
-The user name is provided as `--user <username>` parameter,
-and the command will prompt you for this user's password.
-These credentials are used only for this upload and will in no case be stored on the device.
-:::
-
-<UserContext>
-
-```sh
-sudo tedge cert upload c8y --user "$C8Y_USER"
-```
-
-</UserContext>
+There are a few different ways to get the device certificate to be trusted by Cumulocity, below lists a few common options.
 
-## Creating an MQTT bridge between the device and the cloud
+### Option 1: Using Cumulocity Certificate Authority (Preview) {#cumulocity-certificate-authority}
 
-The connection from the device to the cloud is established using a so-called MQTT bridge:
-a permanent secured bidirectional MQTT connection that forward messages back and forth
-between the two end-points.
+The [Cumulocity Certificate Authority](https://cumulocity.com/docs/device-certificate-authentication/certificate-authority/) feature is currently in the Public Preview phase and hence needs to be enabled for a specific tenant before it can be used. Please read the [Certificate Management Reference Guide](../../../references/certificate-managements#enable-cumulocity-certificate-authority) on how to enable this feature in Cumulocity.
```

Comment:
Broken link

```suggestion
The [Cumulocity Certificate Authority](https://cumulocity.com/docs/device-certificate-authentication/certificate-authority/) feature is currently in the Public Preview phase and hence needs to be enabled for a specific tenant before it can be used. Please read the [Certificate Management Reference Guide](../../../references/certificate-management#enable-cumulocity-certificate-authority) on how to enable this feature in Cumulocity.
```

Reply:
Fixed in 074aaeee648fe3526fe2005d53dc2685524cc991